docs/clay_over_aoi.ipynb

```diff
@@ -0,0 +1,727 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c67aed17-49b0-4cc3-9bcb-6110c05ff809",
+   "metadata": {},
+   "source": [
+    "# How to run clay over custom AOIs\n",
+    "\n",
+    "This script shows in a few simple steps how the clay model can be run for custom AOIs and over custom date ranges.\n",
+    "\n",
+    "## Download and open global list of MGRS tiles"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b11e3a6a-3fa9-4c36-9f02-8991f7b4ec8e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from pathlib import Path\n",
+    "\n",
+    "# The repo home is our working directory.\n",
+    "wd = Path.cwd().parent\n",
+    "os.chdir(wd)\n",
+    "# Ensure data directories exist\n",
+    "Path(\"data/mgrs\").mkdir(exist_ok=True)\n",
+    "Path(\"data/chips\").mkdir(exist_ok=True)\n",
+    "Path(\"data/checkpoints\").mkdir(exist_ok=True)\n",
+    "Path(\"data/embeddings\").mkdir(exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ac5678f0-1f40-42c2-853f-f1905f4b2cdb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-01-12 16:21:52--  https://clay-mgrs-samples.s3.amazonaws.com/mgrs_full.fgb\n",
+      "Resolving clay-mgrs-samples.s3.amazonaws.com (clay-mgrs-samples.s3.amazonaws.com)... 3.5.28.121, 52.217.202.65, 54.231.230.89, ...\n",
+      "Connecting to clay-mgrs-samples.s3.amazonaws.com (clay-mgrs-samples.s3.amazonaws.com)|3.5.28.121|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 13787464 (13M) [binary/octet-stream]\n",
+      "Saving to: ‘data/mgrs/mgrs_full.fgb’\n",
+      "\n",
+      "data/mgrs/mgrs_full 100%[===================>]  13.15M  3.55MB/s    in 4.3s    \n",
+      "\n",
+      "2024-01-12 16:21:57 (3.04 MB/s) - ‘data/mgrs/mgrs_full.fgb’ saved [13787464/13787464]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import geopandas as gpd\n",
+    "\n",
+    "! wget https://clay-mgrs-samples.s3.amazonaws.com/mgrs_full.fgb -O data/mgrs/mgrs_full.fgb"
```

Comment:
Should we mention the original KML file this `mgrs_full.fgb` file was converted from? Assuming that you got it from https://sentinel.esa.int/web/sentinel/missions/sentinel-2/data-products at https://sentinel.esa.int/documents/247904/1955685/S2A_OPER_GIP_TILPAR_MPC__20151209T095117_V20150622T000000_21000101T000000_B00.kml?

Reply:
Good idea, the full sample file is constructed in the [landcover.sh](https://github.com/Clay-foundation/model/blob/main/scripts/landcover.sh) script. There I actually got it from the NASA site but its the same file. I'll link to this in the notebook.